oolab/src/test/java/agh/ics/oop/model/MapDirectionTest.java

```diff
@@ -0,0 +1,24 @@
+package agh.ics.oop.model;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MapDirectionTest {
+    @Test
+    void testNext() {
```

Comment:
dobrym nawykiem jest unikanie w testach przedrostków nazw typu `test`, `check`, `should`, itp, bo to nic nie wnosi. Lepiej nazwać metodę tak by jasno opisywała przypadek, np. `vectorCanBeFollowedByAnother`